ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -295,6 +302,7 @@ ${FIPS_SWITCH} \
 ${KMS_KEY_SWITCH} \
 ${PRIVATE_SWITCH} \
 ${PRIVATE_LINK_SWITCH} \
+${USER_TAGS_SWITCH} \
```

Comment:
We do not need `USER_TAGS_SWITCH. Adding `--tags ${USER_TAGS}` directly.

Reply:
Ack, will update

Reply:
@krishvoor  You forget removing the undefined switch.

Reply:
My bad updated them now